casepro/utils/tests.py

```diff
@@ -86,7 +86,7 @@ def to_json(self):
             MyClass()
         ]
 
-        self.assertEqual(json_encode(data), '["string", "2015-10-09T14:48:30.123456", "bar", {"bar": "X"}]')
+        self.assertEqual(json_encode(data), '["string", "2015-10-09T14:48:30.123456Z", "bar", {"bar": "X"}]')
```

Comment:
Due to updating to latest rapidpro client which provides the date formatting function